### PR TITLE
upgrade bitops to 2.2

### PR DESCRIPTION
**Pull request by mickmcgrath13**

### CODE REVIEW

**Comment by mickmcgrath13** [comment]
this can be tested again now that 2.2.1 is out

File: operations/_scripts/deploy/deploy.sh [hunk 1]
```diff
@@ -57,4 +57,4 @@ docker run --rm --name bitops \
 -e TF_STATE_BUCKET="${TF_STATE_BUCKET}" \
 -e DEFAULT_FOLDER_NAME="_default" \
 -v $(echo $GITHUB_ACTION_PATH)/operations:/opt/bitops_deployment \
-bitovi/bitops:2.1.0
+bitovi/bitops:2.2.0
```
Comment (armab) [inline operations/_scripts/deploy/deploy.sh:60]:
```suggestion
bitovi/bitops:2.2.1
```

**Comment by mickmcgrath13** [comment]
@LeoDiazL test with this repo: https://github.com/bitovi/devops-training-ec2-gha-example

**Comment by LeoDiazL** [comment]
Tests results are OK

**Pull request merged**